db.js

```diff
@@ -547,27 +546,24 @@ exports.init = function (sbot, config) {
     )
   }
 
-  function resetAllIndexes(cb) {
-    const done = multicb({ pluck: 1 })
+  function stopUpdatingIndexes() {
     if (abortLogStreamForIndexes) {
       abortLogStreamForIndexes()
       abortLogStreamForIndexes = null
     }
-    for (const indexName in indexes) {
-      indexes[indexName].reset(done())
-    }
-    done(function onResetAllIndexesDone() {
-      cb()
-      updateIndexes()
-    })
   }
 
-  function restartUpdateIndexes() {
-    if (abortLogStreamForIndexes) {
-      abortLogStreamForIndexes()
-      abortLogStreamForIndexes = null
+  function resumeUpdatingIndexes() {
+    if (abortLogStreamForIndexes) return
+    else indexesStateLoaded.onReady(updateIndexes)
+  }
```

Comment:
These functions are already in the `formats-split` branch but I had to backport them here.